debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/rest/DebeziumMongoDbConnectorResource.java

```diff
@@ -51,6 +62,16 @@ public Connector getConnector() {
         return new MongoDbConnector();
     }
 
+    @Override
+    public MetricsDescriptor getMetrics(String connectorName, MBeanServer mBeanServer)
```

Comment:
mBeanServer is never used here and in no other `Debezium*ConnectorResource#getMetrics()` call. The parameter can be removed here and from the interface. MBeanServer instance is only used inside `MetricsResource`  and I think it can be defined there, maybe even as a static const.

Reply:
I think if you'd like to turn this into `default` method then it wuld be better to pass it as a parameter.

Reply:
@jpechane there is no call to MBeanServer in the implementations. @ani-sha do you think we require it as parameter for adding additional (connector-specific) metrics. Or can we add other ways to specify how addditional metrics can be added? Then it can be safely removed.